api/Trace/Span.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Trace;
+
+interface Span extends SpanStatus, SpanKind
+{
+    function getSpanName(): string;
+    function getContext(): SpanContext;
+    function getParent(): ?SpanContext;
+    function getStartTimestamp(): string;
+    function getEndTimeStamp(): ?string;
+    function getAttributes(): Attributes;
+    function getLinks(): Links;
+    function getEvents(): Events;
+    function getStatus(): SpanStatus;
+
+    /**
+     * Attributes SHOULD preserve the order in which they're set. Setting an attribute with the same key as an existing
+     * attribute SHOULD overwrite the existing attribute's value.
+     * @param string $key
+     * @param bool|int|float|string|array $value Note: the array MUST be homogeneous, i.e. it MUST NOT contain values
+     *                                           of different types.
+     * @return Span Must return $this to allow setting multiple attributes at once in a chain
+     */
+    function setAttribute(string $key, $value): Span;
+
+    /**
+     * @param string $name
+     * @param Attributes|null $attributes
+     * @param string|null $timestamp
+     * @return Span Must return $this to allow setting multiple attributes at once in a chain.
+     */
+    function addEvent(string $name, ?Attributes $attributes = null, ?string $timestamp = null): Span;
+
+    /**
+     * @param SpanContext $context
+     * @param Attributes|null $attributes
+     * @return Span Must return $this to allow setting multiple links at once in a chain.
+     */
+    function addLink(SpanContext $context, ?Attributes $attributes = null): Span;
+
```

Comment:
I think you missed the SetStatus API. https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-tracing.md#set-status

Reply:
Fixed in a recent commit 👍